File: 07-09-2023 1064/1064.c
#include <stdio.h>
 
int main() {
 
    int contador = 6;
    double total = 0, media, valor, contadornumero;
    int i;

    for (i = 0; i < contador; i++) {
        scanf("%lf", &valor);

        if (valor > 0) {
            contadornumero++;
            total += valor;
        }
    }

    media = total / contadornumero;

    printf("%.0lf valores positivos\n", contadornumero);
    printf("%.1lf\n", media);
 
    return 0;
}

File: Iniciante/Pagina 02/1064/1064.py
total = 0
positivo = 0

for i in range(6):
    valor = float(input())

    if valor > 0:
        positivo += 1
        total += valor

media = total / positivo

print("%.0lf valores positivos" %positivo)
print("%.1lf" %media)